docs/tutorial/using-grpc-plugins.mdx

```diff
@@ -218,19 +243,19 @@ func (s *StarwarsService) QueryPeople(ctx context.Context, req *service.QueryPeo
 
 This updates the `main.go` that comes with the initial plugin template in a few ways:
 
-1. We removed resolvers for the old `QueryHello` RPC and a field `nextId` in the `StarwarsService` struct. We won't need it anymore.
+1. We add a constant `swapiBaseURL` that we'll later use to construct the URL for the SWAPI API endpoint.
 2. In the `StarwarsService` struct, we added a field of type `*http.Client`. 
     - This will hold a persistent HTTP client that our endpoints can use for the whole lifetime of the plugin process.
```

Comment:
I would make this a. b. so it maps correctly to 2a and 2b.

Reply:
I added (b) and (a), nesting the lists doesn't work properly in Mintlify markdown renderer

Reply:
That is unfortunate, in that case do you think it makes sense to do. (2b) and (2a) instead of (a) and (b), to be a bit more explicit.

Reply:
done